packages/app-builder-lib/src/util/filename.ts

```diff
@@ -2,8 +2,9 @@
 import * as _sanitizeFileName from "sanitize-filename"
 import * as path from "path"
 
-export function sanitizeFileName(s: string): string {
-  return _sanitizeFileName(s)
+export function sanitizeFileName(s: string, normalizeNfd = false): string {
+  const sanitized = _sanitizeFileName(s)
+  return normalizeNfd ? sanitized.normalize("NFD") : sanitized
```

Comment:
What is `normalize("NFD")`? I don't see it in the API or type definitions: https://github.com/parshap/node-sanitize-filename/blob/master/index.d.ts

Reply:
Thank you for clarifying, and I appreciate the links as well!